app/src/org/commcare/tasks/ProcessAndSendTask.java

```diff
@@ -334,6 +328,7 @@ private void sendForms(FormRecord[] records) throws TaskCancelledException {
                             CommCareApplication.notificationManager().reportNotificationMessage(
                                     NotificationMessageFactory.message(ProcessIssues.RecordFilesMissing), true);
                             FormRecordCleanupTask.wipeRecord(c, record);
+                            results[i] = FormUploadResult.RECORD_FAILURE;

```

Comment:
Question1: When a form file is missing when we try to submit, I think we should be counting this is as a `RECORD_FAILURE`, right? With the code as-is, the result for this form will be `FAILURE` (since that's what all values in the `results` array are initialized to to begin with), which is technically incorrect.

Reply:
That sounds correct to me, but I'm not as fresh on the return codes.